assets/Script/General/NativeSdk.ts

```diff
@@ -183,6 +183,8 @@ export class DefaultNativeSdk {
             await stream.write(content);
             await stream.close();
         } catch (error) {
+            if(error instanceof AbortError)
+                throw error;
```

Comment:
Is AbortError only raised from showFileSavePicker? What about the other two API calls?

Reply:
Only `createWriteable`() should throw it, it also indicates the game wasn't saved to the file.

Reply:
I re-read my old code. I think the `try...catch` here is mostly used for feature detection (in case `showFileSavePicker` is not supported). The transfer save is done purely on the client (via rotating the trade token) so you are correct that this does not need server change. Apologize for giving the incorrect information in our earlier discussion as I haven't touched this code for more than 2 years.

Both `showFileSavePicker` and `createWriteable` can throw "AbortError" and other errors - it's a bit hard to see which error should be propagated and which should be swallowed. It may be better to just propagate all errors in the `showFileSavePicker` case.

Reply:
Yeah, when I said "only createWriteable() should throw it" I meant out of the two other calls. It's correct that the `showFileSavePicker())` function can (and as we already know, does :P) do it too. And no hard feelings. If I were you, I absolutely would do that seeing as there is no feasible way to end up with a saved game if an exception occurs in this function, to my best knowledge, but as a first time contributor in a new project, I didn't want to make changes more invasive than strictly necessary, especially given that there is a fallback present that handles other reasons for failing as the mentioned possibility of the API being unsupported.

Reply:
Do you need anything else to move this forward?

Reply:
No. This PR is good as is. I also want to be transparent that Industry Idle had been in an archived state for a while. My laptop that had a working setup broke a while ago and I haven't got the time to set up a new environment and all the build pipelines (Industry Idle supports Web, Windows, Mac, Linux, Android and iOS, so lots of set up.)

My plan is to find some time to set up the build and bring it out of "Early Access". And I will merge this PR then, but it might take a while.